src/index.html

```diff
@@ -3,11 +3,48 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div class="card" data-qa="card">
+
+      <a href="#" class="card__link link">
+        <img class="link__image" src="./images/imac.jpeg"
+        alt="Imac"
+        >
+      </a>
+
+      <a href="#" class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="card__rating rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="rating__reviews">Reviews: 5</div>
+      </div>
+
+      <div class="card__price price">
+        <div class="price__text">Price:</div>
+        <div class="price__numbers">$2,199</div>
+      </div>
+
+      <a href="#" class="card__button-buy"
+      data-qa="hover">
```

Comment:
<img width="357" alt="attrCheklist" src="https://github.com/mate-academy/layout_product-cards/assets/77551820/ad1ea5c7-8d1e-4bd6-92a1-aa5195ca547c">

Reply:
not fixed, pls pay attention to the difference between good example and your implementation
also pls fix the same issue for all such cases in your project because if a mentor points out a mistake, it means that you need to fix ALL occurrences of the mistake, not only in the place where the comment was left.

Reply:
Not fixed